shell/pages/explorer/resource/detail/configmap.vue

```diff
@@ -0,0 +1,27 @@
+<script setup lang="ts">
+import Loading from '@shell/components/Loading.vue';
+import DetailPage from '@shell/components/Resource/Detail/Page.vue';
+import { useFetch } from '@shell/composables/useFetch';
+import { CONFIG_MAP } from '@shell/config/types';
+import { useStore } from 'vuex';
+
+const store = useStore();
+
+const fetch = useFetch(async() => {
```

Comment:
I was originally using async setup blocks with `<Suspense>` but this was causing one of our e2e tests to fail in CI but not locally. I eventually moved around this by adding the fetch composable.